Softeq.XToolkit.WhiteLabel.Droid/Navigation/FrameNavigationService.cs

```diff
@@ -126,14 +127,16 @@ public void RestoreNavigation()
             {
                 NavigateToExistingViewModel(viewModel);
             }
+
+            return Task.CompletedTask;
         }
 
         public void NavigateToViewModel<TViewModel>(IReadOnlyList<NavigationParameterModel> navigationParameters)
             where TViewModel : IViewModelBase
         {
             var viewModel = _iocContainer.Resolve<TViewModel>();
             viewModel.ApplyParameters(navigationParameters);
-            NavigateToViewModel(viewModel);
+            NavigateToViewModelAsync(viewModel);
```

Comment:
In this case, we need logging about errors in navigation.
As an option: `.FireAndForget(_logger)`